lib/plugins/aws/provider/awsProvider.js

```diff
@@ -243,16 +243,21 @@ class AwsProvider {
                 e.statusCode !== 403 && // Invalid credentials
                 ((e.providerError && e.providerError.retryable) || e.statusCode === 429)
               ) {
+                const nextTryNum = numTry + 1;
+                const exponentialBackOffTime = Math.round(
+                  5000 + nextTryNum * Math.random() * 10000
```

Comment:
Maybe we can simplify, and set retry timeout to something between 4 and 7 seconds.

I wouldn't also add back off, it doesn't seem justified (we do not wait for long going request to finish).

Reply:
@philiiiiiipp have you seen this comment maybe?